subprojects/launcher/src/main/java/org/gradle/launcher/daemon/server/DaemonRegistryUpdater.java

```diff
@@ -47,7 +48,7 @@ public DaemonRegistryUpdater(DaemonRegistry daemonRegistry, DaemonContext daemon
     }
 
     public void onStartActivity() {
-        LOGGER.info("Marking the daemon as busy, address: {}", connectorAddress);
+        LOGGER.info("Marking the daemon as busy, address: {}, token: {}", connectorAddress, Arrays.toString(token));
```

Comment:
Is it a good idea to log the authentication token? I thought that should be treated as a secret, right?